pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -163,12 +164,14 @@ func (h *NetworkUtilsHandler) ConfigureIpv6Forwarding() error {
 }
 
 func (h *NetworkUtilsHandler) IsIpv6Enabled() bool {
-	podIp := os.Getenv("MY_POD_IP")
-	if !netutils.IsIPv6String(podIp) {
-		log.Log.V(5).Info("Since the pod ip is non IPv6, IPv6 is disabled")
-		return false
+	podIps := strings.Split(os.Getenv("MY_POD_IPS"), ",")
+	for _, ip := range podIps {
```

Comment:
as we spoke, lets fallback please to MY_POD_IP if it doesnt exists ?
https://sysdig.com/blog/whats-new-kubernetes-1-17/

Reply:
AFAIK, the oldest version we support is k8s 1.16 and it [has](https://storage.googleapis.com/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/3977/pull-kubevirt-e2e-k8s-1.16/1293427896907468800/artifacts/k8s-reporter/2_pods.log) `status.podIPs`.
So we should be fine and there is no need for a fallback.

Reply:
k8s 1.16 has `status.podIPs` but we cannot create env var from it since the downward api doesn't expose it. Changed the code so it won't depend on `status.podIPs`.